openedx/core/djangoapps/profile_images/tests/test_views.py

```diff
@@ -104,6 +109,15 @@ class ProfileImageUploadTestCase(ProfileImageEndpointTestCase):
     """
     _view_name = "profile_image_upload"
 
+    def check_upload_event_emitted(self):
+        """
+        Make sure we emit a UserProfile event corresponding to the
+        profile_image_uploaded_at field changing.
+        """
+        self.assert_profile_event_emitted(
+            profile_image_uploaded_at={'old_value': u'None', 'new_value': unicode(TEST_UPLOAD_DT)}
```

Comment:
We still return `has_image` as a key in the `profile_image` map from the API. But at the data layer, there's no such field as `has_image` on the profile - it's been replaced by a timestamp field which keeps track of the last uploaded (null if unset) profile image. This allows us to both tell if a user has an image, and to use that timestamp to subvert browser and CDN caches.